bindgen/bindgen.bzl

```diff
@@ -126,6 +129,9 @@ def _rust_bindgen_impl(ctx):
         "LIBCLANG_PATH": libclang_dir,
         "RUST_BACKTRACE": "1",
     }
+    cc_toolchain, feature_configuration = find_cc_toolchain(ctx)
+    _, _, linker_env = get_linker_and_args(ctx, ctx.attr, cc_toolchain, feature_configuration, None)
+    env.update(**linker_env)
```

Comment:
Does this need any special handling in the case of clashes with things we explicitly set? e.g. should we be prepending/appending to the `LD_LIBRARY_PATH` rather than overwriting it?

Reply:
There's that issue, and whether the the system include directories from the cc_toolchain should be added. I'm inclined to leave it as I've written it, and adjust if people have problems with it, since this patch is clearly a step forward.

Reply:
Yeah, sounds reasonable - let's see how we go!